core/src/data/value/error.rs

```diff
@@ -72,6 +72,16 @@ pub enum ValueError {
     #[error("unreachable failure on parsing number")]
     UnreachableNumberParsing,
 
+    #[error("failure to convert {a:#?} to {b:#?} for value:{value:#?}")]
+    ConversionErrorFromDataTypeAToDataTypeB {
+        a: DataType,
+        b: DataType,
+        value: Value,
+    },
+
+    #[error("conflict - unique constraint cannot be used for {0} type")]
+    ConflictDataTypeOnUniqueConstraint(String),
```

Comment:
This is not being used in this PR. you can remove this

Reply:
good catch..   I will remove it.